legate/core/solver.py

```diff
@@ -350,21 +350,21 @@ def compute_launch_shape(
         partitions: dict[PartSym, PartitionBase],
         all_outputs: set[Store],
         unbound_ndim: Optional[int],
-    ) -> Optional[Shape]:
+    ) -> Tuple[Optional[Shape], bool]:
```

Comment:
I would add a comment to this now explaining what the boolean return value is.

Reply:
https://github.com/nv-legate/legate.core/pull/510/commits/595965ec41013f28447c8294b7f8811ff327c763